lib/utils.js

```diff
@@ -85,6 +85,9 @@ var utils = module.exports = {
         if (params.stripe_account) {
           opts.headers['Stripe-Account'] = params.stripe_account;
         }
+        if (params.stripe_version) {
+          opts.headers['Stripe-Version'] = params.stripe_version;
+        }
```

Comment:
Can you add a test case for this to `test/utils.spec.js`? There's some similar ones in there already.